src/TextUI/Help.php

```diff
@@ -47,7 +47,7 @@ public function __construct(?int $width = null, ?bool $withColor = null)
         foreach ($this->elements() as $options) {
             foreach ($options as $option) {
                 if (isset($option['arg'])) {
-                    $this->lengthOfLongestOptionName = max($this->lengthOfLongestOptionName, isset($option['arg']) ? strlen($option['arg']) : 0);
```

Comment:
because of the outer `if (isset($option['arg']))` condition, this nested `isset()` can never be false